src/blueprint.zig

```diff
@@ -335,6 +337,116 @@ pub const Pattern = struct {
 	}
 };
 
+pub const Mask = struct {
+	entries: ListUnmanaged(Entry),
+
+	pub const separator = ',';
+	pub const inverse = '!';
+	pub const tag = '$';
+	pub const property = '@';
+
+	const Entry = struct {
+		inner: Inner,
+		isInverse: bool,
+
+		const Inner = union(enum) {
+			block: struct {typ: u16, data: ?u16},
+			blockTag: Tag,
+			property: Property,
+
+			const Property = enum {transparent, collide, solid, selectable, degradable, viewThrough, allowOres, isEntity};
```

Comment:
I would suggest to extract this generically, otherwise we will need to maintain and update this list.
You can use `std.meta.declarations` to get a list of all the function, and then you just need to check if the result is a bool.